redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/CodeListener.kt

```diff
@@ -15,6 +15,9 @@
  */
 package app.cash.redwood.treehouse
 
+import kotlin.native.ObjCName
+
+@ObjCName("CodeListener", exact = true)
```

Comment:
These ones we might have to eventually go back and prefix with Treehouse or something. There's a few really generic names from this module. I guess we'll see over time.

Reply:
Yep, same with `Widget`.